.travis.yml

```diff
@@ -14,24 +14,28 @@ cache:
   timeout: 600
   directories:
     - $HOME/cache
-
 jobs:
   include:
-    - stage: build third party
-      script:
-        - cd earth_enterprise/src
-        - if [ -f $HOME/cache/third_party.tgz ]; then tar xf $HOME/cache/third_party.tgz; fi
-        - scons -j3 release=1 third_party > build.log
-        - mkdir -p $HOME/cache
-        - tar cfz $HOME/cache/third_party.tgz NATIVE-REL-x86_64 .sconsign.dblite .sconf_temp
-    - stage: build fusion and test
-      script:
-        - cd earth_enterprise/src
-        - if [ -f $HOME/cache/third_party.tgz ]; then tar xf $HOME/cache/third_party.tgz; fi
-        - cd ..
-        - scons -j3 release=1 build > build.log
-        - cd src/NATIVE-REL-x86_64/bin/tests
-        - travis_wait 20 ./RunAllTests.pl
+  - stage: build third party
+    env:
+      - CPP_STD=98
+      - BUILD_TYPE=release
+    script: scripts/ci/build_third_party.sh
+  - stage: build fusion and test
+    env:
+      - CPP_STD=98
+      - BUILD_TYPE=release
+    script: travis_wait 45 scripts/ci/build_fusion_and_test.sh
+  - stage: build third party
+    env:
```

Comment:
it would be great if we can separate the two build into two ci ... Would that be doable ?
That way if C++98 failed only we would know about it, instead of not knowing why the build failed ...

Reply:
This does that.  The C++11 and C++98 run independently and in parallel of each other.  You can tell if it is the C++11 or the C++98 build that failed.  See:

Reply:
The one thing it doesn't do is give you a build matrix in side of the MD file which is what I think you wanted.  The unfortunate thing is with the newer Travis syntax we are using this feature is not yet supported.  We are not using the `matrix` syntax but instead are using the `stage` syntax and I can't find anything that lets you create something like what the `matrix` has for reporting results in a cool looking matrix...  Maybe they will add something where the individual stages can report results.  

Also not that the link above doesn't show the environment variables as I didn't add them until later.  With the environment variables you can more clearly see which stage is running what build.  See later runs for what I'm talking about.

Reply:
Looking at this recent build https://travis-ci.org/google/earthenterprise/builds/369736104 ... the C++ version is listed there.
Ok about not having the `matrix` setup. We can revisit it later.